podcasts/en.lproj/Localizable.strings

```diff
@@ -5244,3 +5244,12 @@
 /* Patron early access to new features marketing message*/
 "feature_marketing_early_access" = "Early access to new features";
 
+/* Upgrade account timeline day 1 of free trial message. */
+"upgrade_account_timeline_day_1" = "Get access to Folders, Shuffle, Bookmarks, and exclusive content";
+
+/* Upgrade account timeline one week before charging message. */
+"upgrade_account_timeline_week_before" = "We’ll notify you about your trial ending.";
+
+/* Upgrade account timeline charging day message. The %1$@ argument is the day of the charge. Ex: You’ll be charged on  September 31th Cancel anytime before.*/
+"upgrade_account_timeline_charging_day" = "You’ll be charged on  %1$@. Cancel anytime before.";
```

Comment:
There's an extra space here (also in the comment above)

```suggestion
"upgrade_account_timeline_charging_day" = "You’ll be charged on %1$@. Cancel anytime before.";
```

Reply:
Fixe here [94f92a6](https://github.com/Automattic/pocket-casts-ios/pull/3322/commits/94f92a61a0293b3778438a6eb00a818ffbcb1ba0)